.azure-devops/templates/trigger-tests.yml

```diff
@@ -121,6 +121,21 @@ jobs:
       pythonVersion: $(python)
       serviceConnection: $(AzureServiceConnection)
 
+- job: 'testHub_job_3'
```

Comment:
Not a blocking comment. As an overarching goal, we should try to be efficient with provisioned infrastructure and re-use as much as we can. Every job we add like this increases agent demand +4 (across all supported python versions),  + independently provisioned infra per job, adding cost.

Reply:
I was planning on putting the endpoint tests into this job too - which would require a separate task. I can check if I can add this to either of the existing tasks right now